app.py

```diff
@@ -29,6 +29,18 @@
 app.wsgi_app = werkzeug.contrib.fixers.ProxyFix(app.wsgi_app)
 
 
+def transform_data(wikis):
+    # Turn itemquality to wp10
+    MODELS = 'models'
+    ITEMQUALITY = 'itemquality'
+    for wiki in wikis:
+        if ITEMQUALITY in wikis[wiki].get(MODELS, {}):
```

Comment:
This is half-generalized and sort of awkward.  We can fix later, but I'd suggest keeping a mapping of models we'll want to transform, { 'itemquality': 'wp10' }.